### ISSUE TITLE
cant install

### ISSUE THREAD

**Issue by kamalbirda** [MSM8916-Samsung/android_device_samsung_a5ltedd#1]
error mounting system while installing (11/11/2015 TWRP 2.8.7.0).

**Comment by fonix232** [comment]
You will either need a recovery compiled for the a5ltedd base, or to modify the update.zip and modify assert from a5ltedd to a5ultexx (current recovery only accepts a5ultexx builds).

@TheWhisp any chance adding either selective platform in recovery, or just dump all possible variants into one recovery?

**Comment by kamalbirda** [comment]
yes...this is after i tried changing the META-INF folder to the one in a5ultexx build...
